### PR TITLE
fix(ci) sync charts files

### LINKED ISSUE #145: CI is always fail because charts files not synced
# charts files not synced
image

# snapshot release always failed
image

### PR DESCRIPTION
**Pull request by bitsf**
fix(ci) sync charts files
close #145

run `make generated-diff-test` and commit files

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -195,22 +195,22 @@ jobs:
...
-      id: import_gpg
-      uses: crazy-max/ghaction-import-gpg@v3
-      with:
-        gpg-private-key: "${{ secrets.GPG_PRIVATE_KEY }}"
-        passphrase: "${{ secrets.GPG_PASSPHRASE }}"
-    - run: make release-test
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        GPG_FINGERPRINT: ${{ steps.import_gpg.outputs.fingerprint }}
+#  # Go Releaser
```
Comment (steven-zou) [inline .github/workflows/tests.yml:198]: Why comment this section?

File: charts/harbor-operator/templates/deployment.yaml [hunk 2]
```diff
@@ -39,40 +39,40 @@ spec:
...
-        - containerPort: 5000
-          name: ready
-          protocol: TCP
         - containerPort: 9443
           name: webhook-server
           protocol: TCP
         - containerPort: 8080
           name: metrics
           protocol: TCP
+        - containerPort: 5000
```
Comment (steven-zou) [inline charts/harbor-operator/templates/deployment.yaml:48]: what's this port?

File: charts/harbor-operator/templates/deployment.yaml [hunk 3]
```diff
@@ -21,16 +21,16 @@ spec:
...
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.namespace
         - name: CONFIGURATION_FROM
           value: 'env:'
         - name: CLASSNAME
           value: {{ .Values.harborClass | quote }}
         - name: LOG_LEVEL
           value: {{ .Values.logLevel | quote }}
+        - name: NAMESPACE
```
Comment (steven-zou) [inline charts/harbor-operator/templates/deployment.yaml:30]: I don't think we need this var in chart

File: charts/harbor-operator/templates/validatingwebhookconfiguration.yaml [hunk 4]
```diff
@@ -47,11 +47,12 @@ webhooks:
 - clientConfig:
     caBundle: Cg==
     service:
-      name: '{{ include "chart.fullname" . }}-webhook-service'
+      name: {{ include "chart.fullname" . | quote }}
```
Comment (steven-zou) [inline charts/harbor-operator/templates/validatingwebhookconfiguration.yaml:50]: different name pattern

**Review by steven-zou — COMMENTED** [review]
commented

File: .github/workflows/tests.yml (hunk 1 above)
Comment (bitsf) [reply .github/workflows/tests.yml:198]: this step is always failed now and it's no need to create a release for each commit

File: charts/harbor-operator/templates/deployment.yaml (hunk 2 above)
Comment (bitsf) [reply charts/harbor-operator/templates/deployment.yaml:48]: all these yaml files are generated

**Review by holyhope — APPROVED** [review]

File: config/default/kustomization.yaml [hunk 5]
```diff
@@ -2,6 +2,8 @@
 #commonLabels:
 #  someName: someValue
 
+namespace: harbor-operator
```
[the remark this answers is not in the document]
Comment (steven-zou) [reply config/default/kustomization.yaml]: @holyhope 

This namespace is added for running `kustomize build config/default | kubectl apply -f -`. I think the override is ok when running `make helm-generate`. This file `config/helm/deployment/kustomization.yaml` has no effect to the `kustomize` templates, right?

cc @bitsf

File: .github/workflows/tests.yml (hunk 1 above)
Comment (bitsf) [reply .github/workflows/tests.yml:264]: it works in release.yml, I think maybe the secrets.GPG_PRIVATE_KEY is not readable in PR CI

**Pull request merged**